swap_meet/item.py

```diff
@@ -1,2 +1,41 @@
+""" Initiate class Item. """
+import uuid as u
+
 class Item:
-    pass
\ No newline at end of file
+    """ Initiate Item class for a specific object in inventory. """
+
+    def __init__ (self, id=None, condition=0.0, age=0):
```

Comment:
For argument's sake, do you think there's a way to dry up this code to deal with raised errors all at once? Not sure if there's an efficient way offhand, but that might be an interesting idea to noodle on.